packages/components/src/test/properties/UniquePropertyValuesSelector.test.tsx

```diff
@@ -294,6 +294,83 @@ describe("UniquePropertyValuesSelector", () => {
     await waitFor(() => expect(container.querySelectorAll(".iui-menu-item").length).to.be.equal(1));
   });
 
+  describe("Date formatting", () => {
+    it(`displays date in valid format when typename is 'shortDate'`, async () => {
+      sinon.stub(Presentation.presentation, "getPagedDistinctValues").resolves({
+        total: 1,
+        items: [{ displayValue: "1410-07-15", groupedRawValues: [""] }],
+      });
+      const datePropertyDescription = {
+        name: "#propertyName",
+        displayLabel: "property",
+        typename: "shortDate",
+        editor: undefined,
+      };
+
+      const { queryByText, container, user } = render(
+        <UniquePropertyValuesSelector property={datePropertyDescription} onChange={() => {}} imodel={testImodel} descriptor={descriptor} />,
+      );
+
+      // open menu
+      const selector = await waitFor(() => queryByText("unique-values-property-editor.select-values"));
```

Comment:
When looking up element for interactions (e.g. clicking, typing) use `getByText` instead of `queryByText`. `getByText` throws if element is not found returns `Element` instead of `Element | null` so there is no need to use `!` when passing that variable to other functions.